tests/unit/feast/service/test_registry.py

```diff
@@ -0,0 +1,95 @@
+"""
+Test feast registry service
+"""
+import pytest
+import pytest_asyncio
+
+from featurebyte.feast.schema.registry import FeastRegistryCreate, FeastRegistryUpdate
+
+
+@pytest.fixture(name="feast_registry_service")
+def feast_registry_service_fixture(app_container):
+    """Feast registry service fixture"""
+    return app_container.feast_registry_service
+
+
+@pytest.fixture(name="feast_feature_store_service")
+def feast_feature_store_service_fixture(app_container):
+    """Feast feature store service fixture"""
+    return app_container.feast_feature_store_service
+
+
+@pytest_asyncio.fixture(name="feast_registry")
+async def feast_registry_fixture(feast_registry_service, feature_list):
+    """Feast registry fixture"""
+    feature_list_model = feature_list.cached_model
+    registry = await feast_registry_service.create_document(
+        data=FeastRegistryCreate(
+            project_name="test_project",
+            feature_lists=[feature_list_model],
+        )
+    )
+    return registry
```

Comment:
This is how we create feast registry (input is expected to be list of deployed feature list models).